store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -260,6 +271,31 @@ public byte[] buildColumnQualifier(final String group, final Properties properti
         return stream.toByteArray();
     }
 
+    /**
+     * Populates the set of groups that aggregate with a time sensitive
+     * aggregation function.
+     */
+    private void populateTimeSensitiveGroups() {
+        // Find the groups that use a time sensitive aggregation function
+        for (final String group : aggregatedGroups) {
+            // Check all type definitions for the group
+            Stream<TypeDefinition> typeStream = StreamSupport.stream(
+                schema.getElement(group).getPropertyTypeDefs().spliterator(), true);
```

Comment:
> true

groups aren't that big, is there any cost to setting up parellel ?

Reply:
It should be fairly cheap to split the data but if there's not going to be that many as you say then probably not worth it

Reply:
groups you'll have 10's maybe a 100.